staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -219,7 +219,7 @@ func (le *LeaderElector) acquire(ctx context.Context) bool {
 		succeeded = le.tryAcquireOrRenew()
 		le.maybeReportTransition()
 		if !succeeded {
-			klog.V(4).Infof("failed to acquire lease %v", desc)
+			klog.Infof("failed to acquire lease %v", desc)
```

Comment:
Won't `failed to acquire lease` be logged repeatedly from the non-leaders for expected cases like "lock is held by %v and has not yet expired", with no way to turn it off, if we remove `V(4)`? Are there error cases in `tryAcquireOrRenew` do we need more logging for that don't already log to error?

Reply:
This is a good point, I havent checked on this PR in a while.  
I wouldnt mind a periodic Infof that rolls up that the last 10 acquires for lease failed, that way its not spammy.

Reply:
Hi @jpbetz   .. Ive updated this to only report once every 10 times which i think means that the message will generally pop up every minute or so, which is i think appropriate.    Let me know what you think ?

Reply:
(obsolete now, removed this change)